ReactWindows/ReactNative/Modules/DeviceInfo/DeviceInfoModule.cs

```diff
@@ -183,29 +174,63 @@ private void OnOrientationChanged(DisplayInformation displayInformation, object
         private void SendUpdateDimensionsEvent()
         {
             Context.GetJavaScriptModule<RCTDeviceEventEmitter>()
-                .emit("didUpdateDimensions", GetDimensions());
+                .emit("didUpdateDimensions", GetAllDimensions());
         }
 
-        private static JObject GetDimensions()
+        private JObject GetAllDimensions()
         {
-            return GetDimensions(DisplayMetrics.GetForCurrentView());
+            var dimensions = new JObject();
+
+            // Default metric for main window
+            // TODO It would make sense to make default actively focused window and not the main in the future
+            var mainView = _registeredViews.Values.FirstOrDefault(v => v.RootView.Dispatcher == CoreApplication.MainView.Dispatcher);
+            var defaultMetric = mainView == null ? DisplayMetrics.Empty : mainView.CurrentDisplayMetrics;
+            dimensions.Add("window", GetDimensions(defaultMetric));
+
+            foreach (var info in _registeredViews.Values)
+            {
+                dimensions.Add($"{info.RootViewTag}", GetDimensions(info.CurrentDisplayMetrics));
```

Comment:
I should have given an example, I was thinking something like:
```
{
    "window": { /* dimensions */ },
    "windows": {
        "key": { /* dimensions */ }
    }
}
```
It doesn't feel quite right to mix the top level structure with arbitrary keys.

Reply:
It may look nicer that way but it would need more checks and processing when accessing from upper RN, RX Layers. So I don't have strong opinion on it, just need to keep RX changes in sync based on this structure ( I already updated RX.UserInterface - https://github.com/Microsoft/reactxp/blob/master/src/native-common/UserInterface.tsx)